proto/viam/app/data/v1/data.proto

```diff
@@ -188,7 +200,13 @@ message DeleteBinaryDataByIDsResponse {
 
 // Annotation specifies the ways in which we can associate annotations with data, e.g. with a text tag or image bounding box
 message Annotation {
-  string tag = 1;
+  string value = 3;
+  Type type = 2;
+}
+
+enum Type {
```

Comment:
Since we're already explicitly returning tags as a separate field, don't need to update Annotation or Type.

Reply:
Taken out